tools/cli/commands/release.js

```diff
@@ -136,10 +141,20 @@ export async function scriptRouter( argv ) {
 				      jetpack release ${ argv.project } release-branch \n`.replace( /^\t+/gm, '' );
 			break;
 		case 'release-branch':
+			argv = await getReleaseVersion( argv );
+			argv.script = `tools/create-release-branch.sh`;
+			argv.scriptArgs = [ argv.project, argv.version ];
+			argv.next = `Finished! Next: 
+				  - Once the branch is pushed, GitHub Actions will build and create a branch jetpack/branch-x.x over at: https://github.com/Automattic/jetpack-production
+				  - jetpack-production/branch-x.x is the built version for release, and will be the branch that is tagged in GitHub and pushed to svn in WordPress.org.
+				  - When changes are pushed to jetpack/branch-x.x, GitHub Actions takes care of building/mirroring to the jetpack-production repo.
```

Comment:
All the references to "jetpack/branch-x.x" and jetpack-production are specific to Jetpack-the-plugin.

The bit about "jetpack-production/branch-x.x" is confusing to me, there's nothing by that name anywhere as far as I know. It seems like you're referring to the "jetpack/branch-x.x" branch in the jetpack-production mirror repo?

Reply:
> All the references to "jetpack/branch-x.x" and jetpack-production are specific to Jetpack-the-plugin.

Ah you're right, nice catch here. I'll update this to be inclusive of all plugins. This copy was from the Jetpack Releases Field Guide page.

>It seems like you're referring to the "jetpack/branch-x.x" branch in the jetpack-production mirror repo?

Yeah this is exactly it. It's basically just "keep an eye on the mirror repo."